badge_server/test_badges.py

```diff
@@ -548,70 +557,45 @@ def assertTargetResponse(self, package_name, *supported_pyversions):
             json_response['dependency_res'],
             {'deprecated_deps': '', 'details': {}, 'status': expected_status})
 
+    def assertImageResponse(self, package_name, left_text):
```

Comment:
How about `assertImageSuccess()` to not override the base class that has a different signature. Likewise for the following classes e.g. `assertImageMissingData()`

I might even have one method for github and one for pypi but that's up to you.

Reply:
I think both suggestions make sense.